azext_iot/common/central_functions.py

```diff
@@ -0,0 +1,149 @@
+# coding=utf-8
```

Comment:
@digimaun FYI this is just a temporary movement of the central functions until we have the new code organization set. Then I'll move this stuff around as part of followup work